### PR TITLE
URLs: Upgrade more http to https

### PR DESCRIPTION
**Pull request by vszakats**

### CODE REVIEW

**Comment by jay** [comment]
Is this part of a series, because we have quite a few of these http => https commits already so I'd like to squash any I can together. When you don't anticipate any more let me know and I'll commit it.

**Comment by vszakats** [comment]
I'm finished for now, no problem at all with squashing it, thank you @jay.

[Just a minor note that https is also supported for example.com, so probably it could be used in a few places in the manual.]

File: docs/FAQ [hunk 1]
```diff
@@ -1402,7 +1402,7 @@ FAQ
   to do "LIST -a" or similar to see them.
 
   The application thus needs to parse the LIST output. One such existing
-  list parser is available at http://cr.yp.to/ftpparse.html  Versions of
+  list parser is available at https://cr.yp.to/ftpparse.html  Versions of
```
Comment (vszakats) [inline docs/FAQ]: Not sure of this one. Maybe update to https://curl.haxx.se ?

File: CHANGES.0 [hunk 2]
```diff
@@ -16359,7 +16359,7 @@ Daniel (17 January 2000):
    progress deal better with larger files and added a "Time" field which shows
    the time spent on the download so far.
  - I'm now using the CVS repository on sourceforge.net, which also allows web
-   browsing. See https://curl.haxx.nu.
+   browsing. See http://curl.haxx.nu.
```
Comment (vszakats) [inline CHANGES.0]: Not sure of this one. Maybe update to https://curl.haxx.se ?

Comment (vszakats) [inline CHANGES.0]: Not sure of this one. Maybe update to https://curl.haxx.se ?

**Comment by jay** [comment]
Looks good but there were a few in the source that were over the 79 length limit. Please run checksrc in the future.
~~~
..\lib\vtls/schannel.c:192:81: warning: Longer than 79 columns
     /* https://msdn.microsoft.com/en-us/library/windows/desktop/aa374716.aspx *
/
..\lib\vtls/schannel.c:410:81: warning: Longer than 79 columns
     /* https://msdn.microsoft.com/en-us/library/windows/desktop/aa375924.aspx *
/
..\lib\vtls/schannel.c:976:81: warning: Longer than 79 columns
     /* https://msdn.microsoft.com/en-us/library/windows/desktop/aa375348.aspx *
/
checksrc: 0 errors and 3 warnings
~~~

Concerning example.com some people build libcurl without SSL so I think it's best if we leave those as http://

Thanks, landed in d49881c.

**Comment by vszakats** [comment]
Sorry for the long lines, I'm in the process training myself to pay attention to that &mdash; I'll be careful next time.

**Pull request closed without merging**